### PR TITLE
LPD-38287 | Reduce flakiness enhancing teardown

### PR DESCRIPTION
**Pull request by jaimelr10**
**Description:** There are some failures caused by the teardown of the picklists. This code adapts the teardown to the apiHelpers.data approach.

### CODE REVIEW

**Comment by jaimelr10** [comment]
ci:test:playwright

**Comment by jaimelr10** [comment]
ci:test:playwright

**Comment by jaimelr10** [comment]
ci:test:playwright

**Comment by jaimelr10** [comment]
ci:test:playwright

**Comment by jaimelr10** [comment]
ci:test:sf

**Comment by jaimelr10** [comment]
ci:test:playwright

**Comment by jaimelr10** [comment]
ci:test:playwright

File: modules/test/playwright/tests/object-web/objectEntries.spec.ts [hunk 1]
```diff
@@ -124,15 +126,15 @@ test.describe('Manage object entries through Page Templates', () => {
 				},
 			});
 
-		objectDefinitions.push(objectDefinition1);
+		apiHelpers.data.push({id: objectDefinition1.id, type: 'objectDefinition'})
```
Comment (ccorreagg) [inline modules/test/playwright/tests/object-web/objectEntries.spec.ts]: I guess we need to remove all the tear-down process we are doing in the afterEach method of test..

**Comment by jaimelr10** [comment]
ci:test:playwright

**Comment by jaimelr10** [comment]
ci:test:playwright

**Comment by jaimelr10** [comment]
ci:stop

**Comment by jaimelr10** [comment]
c:test:sf

**Comment by jaimelr10** [comment]
ci:test:playwright

**Comment by jaimelr10** [comment]
ci:test:sf

**Comment by jaimelr10** [comment]
ci:test:playwright

**Comment by ccorreagg** [comment]
ci:forward:force

**Comment by ccorreagg** [comment]
ci:test:sf

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
ci:test:playwright

**Pull request closed without merging**